README.md

```diff
@@ -198,6 +198,14 @@ require('gitlinker').setup({
         .. "{_A.FILE}"
         .. "#lines-{_A.LSTART}"
         .. "{(_A.LEND > _A.LSTART and (':' .. _A.LEND) or '')}",
+      -- example: https://codeberg.org/bagnaram/dotfiles/src/branch/master/Makefile
+      ["^codeberg%.org"] = "https://codeberg.org/"
+        .. "{_A.USER}/"
+        .. "{_A.REPO}/src/"
+        .. "{_A.REV}/"
+        .. "{_A.FILE}"
+        .. "#L{_A.LSTART}"
+        .. "{(_A.LEND > _A.LSTART and ('-L' .. _A.LEND) and '?display=source' or '')}",
```

Comment:
1. use git commit instead of `master` branch.

2. the `display=source` is before line numbers: `https://codeberg.org/linrongbin16/gitlinker.nvim/src/commit/a570f22ff833447ee0c58268b3bae4f7197a8ad8/README.md?display=source#L2`